### ISSUE TITLE
[Feature Request] Flip colors in color map

### ISSUE THREAD

**Issue by sandner-art** [hako-mikan/sd-webui-cd-tuner#25]
The colors could flip direction with a Flip button, in both horizontal or vertical mode.

**Comment by sandner-art** [comment]
Just to clarify, it would allow to use of the same color gradient without the need of manual editing of color date. It would be useful to adjust diffusion for a quick generation of variants.

**Comment by hako-mikan** [comment]
Does that mean it simply rearranges the order of the colors?
For example, if we have colors like 
5,5,5; 0,5,5; 5,0,0; -5,-5,-5, 
pressing the flip button would change them to
-5,-5,-5; 5,0,0; 0,5,5; 5,5,5?

**Comment by sandner-art** [comment]
Yes, this is what I mean. If you have more colors it is hard to do manually.
